src/ploomber/util/default.py

```diff
@@ -456,6 +460,12 @@ def find_root_recursively(starting_dir=None,
         starting_dir=starting_dir,
     )
 
+    if root_by_pipeline == _filesystem_root():
+        raise DAGSpecInvalidError(
+            'pipeline.yaml cannot be in the filesystem root. '
+            'Please add it inside a directory like '
+            'project-name/pipeline.yaml. ')
```

Comment:
please use the filename to determine the error message, since it might change. 

```python
f'project-name/{filename}'
```

Reply:
Thanks for the review @edublancas  ! Made the changes